components/suggest/src/suggest.udl

```diff
@@ -127,6 +127,12 @@ interface SuggestProviderConfig {
     );
 };
 
+enum InterruptKind {
+    "Read",
+    "Write",
+    "Everything",
```

Comment:
Absolute nitpick: What do you think of `ReadWrite` instead of `Everything` as a name? 😅

This new API is so elegant!